src/tbots.py

```diff
@@ -204,11 +226,13 @@
             sys.exit(1)
         bazel_arguments += ["-pb deploy_robot_software.yml"]
         bazel_arguments += ["--hosts"]
-        platform_ip = "0" if args.platform == "NANO" else "5"
-        bazel_arguments += [f"192.168.{platform_ip}.20{id}" for id in args.flash_robots]
-        bazel_arguments += ["-pwd", args.pwd]
+        bazel_arguments += [
+            NetworkConstants.get_ip_address(id, RobotPlatform(args.platform))
```

Comment:
```suggestion
            str(NetworkConstants.get_ip_address(id, RobotPlatform(args.platform)))
```
We should cast it to `str` since `get_ip_address` is an IPv4Address